### PR TITLE
Add whole-body-controllers to the superbuild

### PR DESCRIPTION
**Pull request by gabrielenava**
I added [whole-body-controllers]() to the superbuild and removed [WBIToolboxControllers](). Tested on Ubuntu 16.04.

### CODE REVIEW

**Comment by gabrielenava** [comment]
I think it make sense. But if we move the repo to robotology this PR must be updated (now the superbuild is pointing `robotology-playground`.

File: cmake/Buildwhole-body-controllers.cmake [hunk 1]
```diff
@@ -8,9 +8,9 @@ include(FindOrBuildPackage)
 find_or_build_package(WBToolbox QUIET)
 find_or_build_package(qpOASES QUIET)
 
-ycm_ep_helper(WBIToolboxControllers TYPE GIT
+ycm_ep_helper(whole-body-controllers TYPE GIT
               STYLE GITHUB
-              REPOSITORY robotology-playground/WBI-Toolbox-controllers.git
+              REPOSITORY robotology-playground/whole-body-controllers.git
```
Comment (traversaro) [inline cmake/Buildwhole-body-controllers.cmake]: This can be moved to `robotology` now: https://github.com/robotology/whole-body-controllers/issues/22 .

**Comment by traversaro** [comment]
Let me know when the announcement is ready, so I can merge the PR.

**Comment by gabrielenava** [comment]
Done here: https://github.com/robotology/QA/issues/288

**Pull request merged**